Resources/Prototypes/Entities/Structures/Storage/filing_cabinets.yml

```diff
@@ -62,61 +58,55 @@
           SheetSteel1:
             min: 1
             max: 2
+  - type: StaticPrice
+    price: 80
   - type: Construction
     graph: FilingCabinet
+
+- type: entity
+  name: filing cabinet
+  parent: BaseFilingCabinet
+  id: filingCabinet
+  components:
+  - type: Storage
```

Comment:
```suggestion
```
Any reason you are overwriting storage again?

Reply:
no